src/packages/v4/views/V4ProjectDashboard/hooks/useV4ProjectHeader.ts

```diff
@@ -0,0 +1,69 @@
+import { useProjectMetadataContext } from 'contexts/ProjectMetadataContext'
+import { BigNumber } from 'ethers'
+import { useGnosisSafe } from 'hooks/safe/useGnosisSafe'
+import { useProjectTrendingPercentageIncrease } from 'hooks/useProjectTrendingPercentageIncrease'
+import { SubtitleType, useSubtitle } from 'hooks/useSubtitle'
+import { GnosisSafe } from 'models/safe'
+import { useRouter } from 'next/router'
+
+export interface ProjectHeaderData {
+  title: string | undefined
+  subtitle: { text: string; type: SubtitleType } | undefined
+  domain: string | undefined
+  handle: string | undefined
+  projectId: number | undefined
+  owner: string | undefined
+  payments: number | undefined
+  totalVolume: BigNumber | undefined
+  last7DaysPercent: number
+  gnosisSafe: GnosisSafe | undefined | null
+  archived: boolean | undefined
+  createdAtSeconds: number | undefined
+  chainName: string
+}
+
+export const useV4ProjectHeader = (): ProjectHeaderData => {
+  const router = useRouter()
+  const { projectMetadata, projectId } = useProjectMetadataContext()
+  // const {
+  //   handle,
+  //   projectOwnerAddress,
+  //   totalVolume,
+  //   trendingVolume,
+  //   paymentsCount,
+  //   createdAt,
+  // } = useContext(V2V3ProjectContext) -> How it looks on v2v3
+
+  const handle = 'placeholderData'
+  const projectOwnerAddress = 'placeholderData'
+  const totalVolume = BigNumber.from(100)
```

Comment:
NOOOOOO bignumber in V4. Sorry, forgot to mention that. BigInt only https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt

```suggestion
  const totalVolume = 100n
```

Reply:
Oh ok, got it